.circleci/config.yml

```diff
@@ -181,6 +202,20 @@ workflows:
     jobs:
       - format_flutter
       - analyze_flutter
+      - pub_analysis:
+          matrix:
+            parameters:
+              plugin_threshold: [
+                  # should be in format <plugin_name>:<threshold>
+                  "amplify_analytics_pinpoint:10",
+                  "amplify_api:20",
```

Comment:
I believe I can decrease this now that https://github.com/aws-amplify/amplify-flutter/pull/1404 has merged. I will update with main and look to decrease it.